script/proptest-regressions/syscalls/mod.txt

```diff
@@ -0,0 +1,7 @@
+# Seeds for failure cases proptest has generated in the past. It is
```

Comment:
I think that this file is generated by proptest in your processing of refactoring. Could you please double check whether we have a unit test of this case?